extensions/amp-3q-player/0.1/validator-amp-3q-player.protoascii

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-3q-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-3q-player extension .js script"
+  satisfies: "amp-3q-player extension .js script"
+  mandatory_parent: "HEAD"
+  requires: "amp-3q-player"
+  unique: true
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-3q-player"
+    dispatch_key: true
+  }
+  attrs: { name: "nonce" }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-3q-player-(latest|0\\.1)\\.js"
+  }
+  attrs: {
+    name: "type"
+    value: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-3q-player.html"
+}
+tags: {  # <amp-3q-player>
+  html_format: AMP
+  tag_name: "AMP-3Q-PLAYER"
+  satisfies: "amp-3q-player"
+  requires: "amp-3q-player extension .js script"
+  attrs: {
+    name: "data-id"
+    mandatory: true
+  }
+  attr_lists: "extended-amp-global"
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-3q-player.html"
```

Comment:
ampproject recently changed the urls, while this will redirect to the new one, lets just point to the new location instead, `https://www.ampproject.org/docs/reference/components/media/amp-3q-player`.